pyaerocom/climatology_config.py

```diff
@@ -0,0 +1,46 @@
+from pydantic import BaseModel, ValidationError, field_validator
+
+from typing import Literal
+
+from pyaerocom import const
+
+
+class ClimatologyConfig(BaseModel):
+    """
+    Holds the configuration for the climatology
+
+    Attributes
+    -------------
+    start : int, optional
+        Start year of the climatology
+    stop : int, optional
+        Stop year of the climatology
+    resample_how : str, optional
+        How to resample the climatology. Must be mean or median.
+    freq : str, optional
+        Which frequency the climatology should have
+    mincount : dict, optional
```

Comment:
I'd suggest `min_count`. Also maybe worth building a `field_validator` for this? Or possibly a child class of `TypedDict`